src/linear_map.rs

```diff
@@ -403,6 +404,7 @@ where
     }
 }
 
+#[derive(Clone)]
```

Comment:
This should be a manual `Clone` implementation. No need for `Clone` bounds on any of the type parameters.